packages/ui/components/NavBar.vue

```diff
@@ -87,8 +87,17 @@ export default {
       deep: true,
     },
   },
+  mounted() {
+    window.addEventListener('resize', this.resizeHandler)
+  },
+  beforeDestroy() {
+    window.removeEventListener('resize', this.resizeHandler)
+  },
   methods: {
     capitalizeFirstLetter,
+    resizeHandler() {
+      this.closeMenu()
```

Comment:
Could we close the menu only when the resize changes in the breakpoint?
```suggestion
      // in data
      previousSize: window.screen.availWidth,
   
     // inside resizeHandler
      const currentWidth = e.target.screen.availWidth
      if (
        (this.previousSize >= 850 && currentWidth <= 850) ||
        (this.previousSize <= 850 && currentWidth >= 850)
      ) {
        // close menu only if resize crosses the breakpoint
        this.closeMenu()
      }
      this.previousSize = currentWidth
```